### DISCUSSION TITLE
Does the default model support GPU? How can i use GPU to synthesize voice?

### DISCUSSION THREAD

**Discussion by thehzzz** [coqui-ai/TTS#2083] [General Q&A]
Does the default model support GPU? How can i use GPU to synthesize voice?

**Comment by domcross** [comment]
Assuming you are using command `synthesize` you can pass parameter `--use_cuda=1` to enable GPU usage. Your Python environment must have a CUDA-enabled PyTorch-library, though.

**Reply by thehzzz** [reply]
Thank you very much
